### PR TITLE
Set room config and invite users to group chat

### PR DESCRIPTION
**Pull request by burmajam**
I've also added optional params for Romeo.Connection so we can provide name to it from supervisor

### CODE REVIEW

File: lib/romeo/connection.ex [hunk 1]
```diff
@@ -45,14 +45,14 @@ defmodule Romeo.Connection do
     * `:timeout` - Connect timeout in milliseconds (default: `#{@timeout}`);
     * `:transport` - Transport handles the protocol (default: `#{@default_transport}`);
   """
-  def start_link(opts) do
+  def start_link(opts, srv_opts \\ []) do
```
Comment (scrogson) [inline lib/romeo/connection.ex:48]: Let's rename `opts` to `config` and `srv_opts` to `opts`.

File: lib/romeo/stanza.ex [hunk 2]
```diff
@@ -178,6 +184,37 @@ defmodule Romeo.Stanza do
     iq(to, "get", xmlel(name: "query", attrs: [{"xmlns", ns_disco_items}]))
   end
 
+  def set_room_config(to, config) do
+    fields = config
```
Comment (scrogson) [inline lib/romeo/stanza.ex:188]: Fix up the indention here like so:

``` elixir
fields =
  config
  |> Map.merge(...)
  |> Enum.map(...)
  |> Enum.reverse()
```

File: lib/romeo/stanza.ex [hunk 3]
```diff
@@ -178,6 +184,37 @@ defmodule Romeo.Stanza do
...
+          name: "x", 
+          attrs: [{"xmlns", ns_data_forms}, {"type", "submit"}],
+          children: fields
+        )
+    ]))
+  end
+
+  def invite(room, invitees) do
+    children = invitees
+                |> Enum.map(fn invitee -> xmlel(name: "invite", attrs: [{"to", invitee}]) end)
```
Comment (scrogson) [inline lib/romeo/stanza.ex:206]: Same indention as above.

**Comment by scrogson** [comment]
Hi @burmajam thanks for putting this together. I've made some comments and it looks like there is probably a conflict that needs to be resolved.

Sorry for the late response.

**Comment by arpunk** [comment]
@scrogson What is the status of this PR?

**Pull request closed without merging**